dcm4ceph-core/src/main/resources/ceph_defaults.properties

```diff
@@ -8,7 +8,7 @@ patientID=""
 
 # Patient's Birth Date (0010,0030) 2 Birth date of the patient.
 # Use ISO format dd-MM-yyy 
-patientDOB=
+patientDOB=0000-01-01
```

Comment:
Do not use a fake default for DOB. Better to keep Null, or not add DOB at all, if DOB is unk. Same with SID and SOD below.

Reply:
Rolled back